tests/s2_inference/embeddings_reference/embeddings_open_clip_python_3_8.json

```diff
@@ -1 +1 @@
-0.0908920019865036, 0.2777882516384125, -0.035417359322309494, 0.11340413987636566, 0.5041928291320801, -0.20269547402858734, 0.35347747802734375, -0.07602190226316452, -0.08942906558513641, -0.1760968416929245, 0.12471196055412292, -0.28563088178634644, 0.3064865171909332, 0.23779144883155823, -0.4003244936466217, -0.2329786717891693, -0.4280110001564026, 0.09755830466747284, 0.2612250745296478, -0.12004019320011139, 0.31127703189849854, 0.16079673171043396, -0.13609814643859863, -0.03735960274934769, 0.1792931854724884, 0.18144862353801727, 0.07729245722293854, 0.313904345035553, 0.060350000858306885, -0.2214396893978119, -0.10060135275125504, -0.14822377264499664, -0.15594244003295898, -0.060674652457237244, -0.07779013365507126, -0.14744551479816437, -0.04703781008720398, -0.34976211190223694, 0.5477346777915955, -0.31047317385673523, -0.09369033575057983, 0.2275630086660385, -0.4484075903892517, -0.0043919384479522705, -0.09938926994800568, -0.598154604434967, 0.2719945013523102, -0.23185338079929352, -0.07749907672405243, -0.3305845260620117, -0.10934503376483917, 0.13795456290245056, -0.04886232689023018, 0.1929512768983841, -0.14878660440444946, -0.039170295000076294, 0.09150396287441254, -0.020953122526407242, -0.06463529169559479, -0.06815842539072037, 1.3180593252182007, 0.06455932557582855, -0.005628447979688644, 0.11132839322090149, 0.10763922333717346, 0.14827656745910645, -0.014446604996919632, 0.1549694538116455, 0.8952656388282776, -0.19357851147651672, 0.020150095224380493, 0.20482642948627472, 0.26591479778289795, 0.4972236156463623, -0.3578789234161377, -0.07334323227405548, -0.04157279431819916, -0.04042378440499306, -0.025470316410064697, -5.036540985107422, 0.0771903544664383, 0.23993131518363953, 0.23642313480377197, 0.10409143567085266, -0.1911507248878479, 0.015132710337638855, -0.06272543966770172, 0.7990829944610596, -0.07715411484241486, -0.12544754147529602, -0.16686873137950897, 0.3222806453704834, -0.29819729924201965, -0.12677013874053955, 0.30761465430259705, -0.06962472945451736, -0.30692076683044434, -0.16202221810817719, 0.10108188539743423, 0.19411630928516388, 0.34583571553230286, 0.29822617769241333, -0.013717830181121826, 0.2638685703277588, -0.486847460269928, 0.05100366100668907, 0.09965290874242783, 0.4005868434906006, -0.4952826499938965, 0.07037579268217087, 0.23237833380699158, 0.06910058856010437, -0.0832923948764801, 0.08270321786403656, 0.00014057010412216187, -0.2475355863571167, 0.36599504947662354, 0.18246735632419586, -0.32911813259124756, -0.10806237906217575, 0.3398933410644531, -0.23570489883422852, -0.2435854822397232, -0.09250035136938095, 0.3117081820964813, 0.12970593571662903, -0.4478328824043274, 0.3289231061935425, 0.2141920030117035, -0.007208868861198425, 0.06782802939414978, -0.019435983151197433, 1.0774712562561035, -0.9808105826377869, -0.1723078191280365, 0.22659990191459656, -0.020131424069404602, 0.1446133255958557, -0.06365127116441727, 0.19897186756134033, 0.07798860222101
```

Comment:
nit. we add lines to those json files to the diff makes more sense in the future

Reply:
i did just that, idk why it shows a completely new file. maybe something to do with a huge number of additions?